sympy/solvers/ode/single.py

```diff
@@ -852,5 +852,80 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [Eq(lhs, C2 + x), Eq(lhs, C2 - x)]
 
 
+class Lienard(SinglePatternODESolver):
+    r"""
+    The general form of the Lienard ODE:
+
+    .. math :: y''(x) + f(x)*y'(x) + y(x)
+
+    We solve using the fact that all linear second order homogeneous ODEs
+    can be transformed into first order ODEs of Riccati type.
+    Here we solve using the transformation as:
+
+    .. math :: b(x) = y'(x)/y(x)
+
+    .. math :: y''(x) = b'(x)*y(x) + y(x)*b(x)**2
+
+    On substituting,
+
+    .. math :: b'(x) + b(x)**2 + f(x)*b(x) + 1
+
+    So this new nonlinear first order ODE in b(x) is obtained, further we can try to find b(x)
+    by solving this ode using dsolve. Later, we can find y(x) by:
+
+    .. math :: y(x) = exp(Integral(b(x),x))
+
+
+    Examples
+    ========
+
+    >>> from sympy import Function, dsolve, Symbol
+    >>> f = Function('f')
+    >>> x = Symbol('x')
+
+    >>> eq = f(x).diff(x, x) + 2/x*f(x).diff(x) + f(x)
+    >>> sol = dsolve(eq, hint='lienard',simplify=False)
+    >>> print(sol)
+    Eq(f(x), exp(C2)/(x*sqrt(tan(C1 + x)**2 + 1)))
+
+    >>> eq = f(x).diff(x,2) + x*f(x).diff(x) + f(x)
+    >>> sol = dsolve(eq, hint='lienard',simplify=False)
+    >>> print(sol)
+    Eq(f(x), exp(x**2*(-C1**2 - 1)/2 + x**4*(C1**2/2 + (-C1**2 - 1)*(6*C1**2 + 1)/6
+    + 1/6)/4 + x**6*(-C1**2*(-3*C1**2 - 1)/15 - 7*C1**2/60 + (-36*C1**2 - 5)*(-C1**2 - 1)/120
+    + (-C1**2 - 1)*(-16*C1**2*(-3*C1**2 - 1) - 36*C1**2 + 8*(-9*C1**2 - 1)*(-C1**2 - 1) - 5)/120 - 1/60)/6
+    + C2 + C1*x + C1*x**3*(2*C1**2 + 1)/6 + C1*x**5*(-12*C1**2 + 8*(-3*C1**2 - 1)*(-C1**2 - 1) - 5)/120 + O(x**7)))
```

Comment:
I think that we should stop `dsolve` from returning series solutions like this. This is not a good example for the docstring. We don't need two examples though so this can just be removed.

Reply:
Sure I will remove it.
Thanks.